core/src/core-api/dom-elements.js

```diff
@@ -0,0 +1,56 @@
+/** @namespace */
+class LuigiElementsManager {
+  /**
+   * Use the functions to get important DOM elements.
```

Comment:
```suggestion
   * Use these functions to get DOM elements.
```
I would remove `important` :) I think that all of them are :D